scripts/flitcli/flit_bisect.py

```diff
@@ -2210,7 +2222,7 @@ def main(arguments, prog=None):
     '''
 
     if elf is None:
-        print('Error: pyelftools is not installed, bisect disabled',
+        print('Error: binutils is not installed, bisect disabled',
```

Comment:
Perhaps it is possible to use either one.  However, that's annoying to deal with.  I would probably prefer simplicity rather than utilizing two alternatives.